src/index.ts

```diff
@@ -21,33 +22,59 @@ export class TestxServer {
   private schema: string;
   private server: Server;
   private serverUrl: string;
+  private expressServer: Express;
   private dbConnection: Knex;
 
   constructor(schema: string) {
     this.schema = schema;
   }
 
   public async start() {
```

Comment:
I would still put the bootstrap as part of the start function, this will keep the API simple because the user could just use the start method or give the flexibility to the user to bootstrap the server separetlly
```suggestion
   public async start() {
     await this.bootstrap();
```

Reply:
Nice @b1zzu :blush: